bcipy/tasks/rsvp/calibration/inter_sequence_feedback_calibration.py

```diff
@@ -0,0 +1,176 @@
+            display=self.window,
+            parameters=self.parameters,
+            clock=self._task.experiment_clock)
+
+        self.feedback_line_color = self.parameters['feedback_line_color']
+
+    def execute(self):
+        self.logger.debug(f'Starting {self.name()}!')
+        run = True
+
+        # Check user input to make sure we should be going
+        if not get_user_input(self.rsvp, self.wait_screen_message,
+                              self.wait_screen_message_color,
+                              first_run=True):
+            run = False
+
+        # Begin the Experiment
+        while run:
+
+            # Get random sequence information given stimuli parameters
+            (ele_sti, timing_sti,
+             color_sti) = random_rsvp_calibration_seq_gen(
+                 self.alp,
+                 num_sti=self.num_sti,
+                 len_sti=self.len_sti,
+                 timing=self.timing,
+                 is_txt=self.is_txt_sti,
+                 color=self.color)
+
+            (task_text, task_color) = get_task_info(self.num_sti,
+                                                    self._task.task_info_color)
+
+            # Execute the RSVP sequences
+            for idx_o in range(len(task_text)):
+
+                # check user input to make sure we should be going
+                if not get_user_input(self.rsvp, self.wait_screen_message,
+                                      self.wait_screen_message_color):
+                    break
+
+                if self.enable_breaks:
+                    pause_calibration(self.window, self.rsvp, idx_o,
+                                      self.parameters)
+
+                # update task state
+                self.rsvp.update_task_state(
+                    text=task_text[idx_o],
+                    color_list=task_color[idx_o])
+
+                # Draw and flip screen
+                self.rsvp.draw_static()
+                self.window.flip()
+
+                # Get height
+                self.rsvp.sti.height = self.stimuli_height
+
+                # Schedule a sequence
+                self.rsvp.stim_sequence = ele_sti[idx_o]
+
+                # check if text stimuli or not for color information
+                if self.is_txt_sti:
+                    self.rsvp.color_list_sti = color_sti[idx_o]
+
+                self.rsvp.time_list_sti = timing_sti[idx_o]
+
+                # Wait for a time
+                core.wait(self._task.buffer_val)
+
+                # Do the sequence
+                last_sequence_timing = self.rsvp.do_sequence()
+
+                # Write triggers for the sequence
+                _write_triggers_from_sequence_calibration(
+                    last_sequence_timing, self._task.trigger_file)
+
+                # TODO implement feedback decision maker
+                position = self._get_feedback_decision()
+                timing = self.visual_feedback.administer(position=position)
```

Comment:
It looks like this is the only code that differs from the calibration.py execute method. We could add a hook in the Calibration class to call a present_feedback method that could be overridden or externally provided. Then this code could be executed in the callback and the _task.execute method could be used as is.